src/snowflake/snowpark/_internal/utils.py

```diff
@@ -297,6 +298,15 @@ def normalize_path(path: str, is_local: bool) -> str:
     return f"'{path}'"
 
 
+def warn_session_config_update_in_multithreaded_mode(config) -> None:
+    if threading.active_count() > 1:
+        logger.warning(
+            "You might have more than one threads sharing the Session object trying to update "
+            f"{config}. This is currently not thread-safe and may cause unexpected behavior. "
```

Comment:
the sense here sounds little bit wired, and updating the config its-self is thread safe, it just make the other operations unsafe. maybe say updating f"{config}" while other tasks are running can potentially cause unexpected behavior.

Reply:
updated